printer_mods/Zen3D/V0_Handle/README.md

```diff
@@ -0,0 +1,13 @@
+# Voron0 Handle for 3mm panels
+
+## print settings:
+
+- Support required, or use Voron0Handle_supported.stl (layer hight 0.2mm)
```

Comment:
Please change `hight` to `height`.

Reply:
Thanks for the suggestion! I made another version: Handle_v2.stl I apply after I try.